pkgs/applications/emulators/mame/default.nix

```diff
@@ -61,7 +60,8 @@ stdenv.mkDerivation rec {
     "USE_SYSTEM_LIB_FLAC=1"
     "USE_SYSTEM_LIB_GLM=1"
     "USE_SYSTEM_LIB_JPEG=1"
-    "USE_SYSTEM_LIB_LUA=1"
+    # https://www.mamedev.org/?p=523
+    # "USE_SYSTEM_LIB_LUA=1"
```

Comment:
```suggestion
    "USE_SYSTEM_LIB_LUA=0"
```

Reply:
Upstream recommend to remove the flag instead of setting it to 0.